pkg/reconciler/route/controller.go

```diff
@@ -90,6 +93,10 @@ func NewControllerWithClock(
 		impl.EnqueueLabelOfNamespaceScopedResource(
 			serving.RouteNamespaceLabelKey, serving.RouteLabelKey)))
 
+	ingressInformer.Informer().AddEventHandler(controller.HandleAll(
+		impl.EnqueueLabelOfNamespaceScopedResource(
+			serving.RouteNamespaceLabelKey, serving.RouteLabelKey)))
```

Comment:
Can't this just be `EnqueueControllerOf`?

Reply:
+1